content/en/real_user_monitoring/reactnative/integrated_libraries.md

```diff
@@ -0,0 +1,28 @@
+---
+title: React Native Integrated Libraries
+kind: faq
+description: Collect RUM data from your React Native projects.
+further_reading:
+- link: https://github.com/DataDog/dd-sdk-reactnative
+  tag: GitHub
+  text: dd-sdk-reactnative Source code
+- link: https://www.datadoghq.com/blog/react-native-monitoring/
+  tag: Blog
+  text: Monitor your React Native applications
+- link: real_user_monitoring/explorer/
+  tag: Documentation
+  text: Learn how to explore your RUM data
+
+---
+## RUM Integrations
+
+### Integrate with ReactNavigation
+Datadog provides automatic integration for the [ReactNavigation](https://reactnavigation.org/) API (minimum supported version is `react-navigation/native@5.6.0`). Add the following in your source code:
+```typescript
+    const navigationRef:React.RefObject<NavigationContainerRef> = React.createRef();
+    // ...
+    <NavigationContainer ref={navigationRef} onReady={
+        ()=>{DdRumReactNavigationTracking.startTrackingViews(navigationRef.current)}}>
+        // ...
+    </NavigationContainer>
+```
```

Comment:
I think this section would need a bit more information, as users also need to install the `@datadog/mobile-react-navigation` package as well: https://www.npmjs.com/package/@datadog/mobile-react-navigation

Shall we copy/paste the complete documentation on this package (from https://www.npmjs.com/package/@datadog/mobile-react-navigation)?

Reply:
We should also add documentation for react-native-navigation: https://www.npmjs.com/package/@datadog/mobile-react-native-navigation

Reply:
I'm ok with copying and pasting the documentation from the page you linked, but will we have to worry if they make any updates on their end?

Reply:
I've added the info. Let me know what you think!

Reply:
@louiszawadzki Oh, I see! The page I added includes a section that was already there called RUM Integrations. Is it a problem if this page ([https://github.com/DataDog/documentation/pull/17810/files#diff-fe468d1914684201319733951903f63f7f10632a433abb3b343c702f4f0b4fcc](https://github.com/DataDog/documentation/pull/17810/files#diff-fe468d1914684201319733951903f63f7f10632a433abb3b343c702f4f0b4fcc)) is slightly different from https://github.com/DataDog/dd-sdk-reactnative/blob/develop/packages/react-navigation/README.md? Or should we make sure they are exactly the same?

Reply:
@rtrieu they can be different, no problems :)